package.json

```
@@ -164,7 +164,7 @@
     "vscode:prepublish": "yarn lint && rm -rf build/ && yarn compile:prod",
     "compile": "esbuild src/extension.ts --bundle --sourcemap --platform=node --target=node14 --external:vscode  --outfile=build/extension.js",
     "compile:prod": "yarn run compile --minify --define:process.env.NODE_ENV=\\\"production\\\"",
-    "lint": "yarn type-check && eslint src",
+    "lint": "yarn type-check && eslint src --fix",
```

Please, leave the `lint` as it was before, and create a second command. 
```suggestion
    "lint": "yarn type-check && eslint src",
    "lint:fix": "yarn lint --fix",
```
Why?
Because we want the `yarn lint` command to break the build if something is wrong. 
When it is running on a CI pipeline, the `--fix` command will make the task pass, but nothing will be fixed there.